main.go

```diff
@@ -32,3 +40,65 @@ func main() {
 		panic(err.Error())
 	}
 }
+
+func getIPAMClient(ctx context.Context, root settings.Source) (*ipamfetcher.Client, error) {
+	ipsPrefixedEnv := &settings.PrefixSource{
+		Source: root,
+		Prefix: []string{"IPS"},
+	}
+	subnetsPrefixedEnv := &settings.PrefixSource{
+		Source: root,
+		Prefix: []string{"SUBNETS"},
+	}
+	customersPrefixedEnv := &settings.PrefixSource{
+		Source: root,
+		Prefix: []string{"CUSTOMERS"},
+	}
+	device42IPsClientComponent := &ipamfetcher.Device42ClientComponent{}
```

Comment:
thought about using the `New` constructor pattern here, but I've noticed it's not too common in our repos